tests/validators/blocks/test_primary_person_list_collector.py

```diff
@@ -62,20 +42,18 @@ def test_primary_person_invalid_list_collector_non_radio():
     assert expected_errors == validator.validate()
 
 
-def test_invalid_primary_person_list_collector_with_no_add_option():
-    filename = "schemas/invalid/test_invalid_primary_person_list_collector_bad_answer_value.json"
-
+def test_invalid_primary_person_list_collector_with_no_add_or_edit_answer_action():
+    filename = "schemas/invalid/test_invalid_primary_person_list_collector_no_add_edit_action.json"
     questionnaire_schema = QuestionnaireSchema(_open_and_load_schema_file(filename))
-    validator = PrimaryPersonListCollectorValidator(
-        questionnaire_schema.get_block("primary-person-list-collector"),
-        questionnaire_schema,
-    )
+    block = questionnaire_schema.get_block("primary-person-list-collector")
+
+    validator = PrimaryPersonListCollectorValidator(block, questionnaire_schema)
 
     expected_errors = [
         {
-            "message": validator.NON_EXISTENT_PRIMARY_PERSON_LIST_COLLECTOR_ANSWER_VALUE,
+            "message": "AddOrEditAnswerForListItem action not found in main block",
```

Comment:
This should reference the class variable to prevent having to update copy in multiple places.